lib/lnc.ts

```diff
@@ -78,6 +78,38 @@ export default class LNC {
         );
     }
 
+    get status() {
+        return (
+            this.wasm &&
+            this.wasm.wasmClientStatus &&
+            this.wasm.wasmClientStatus()
+        );
+    }
+
+    get expiry() {
+        return (
+            this.wasm &&
+            this.wasm.wasmClientGetExpiry &&
+            new Date(this.wasm.wasmClientGetExpiry() * 1000).toLocaleString()
+        );
+    }
+
+    hasPerms(permission: string) {
```

Comment:
I'd put this function below the all of the `get` properties to keep the file better organized.